typing/typedtree.mli

```diff
@@ -267,7 +267,8 @@ and class_expr_desc =
                   (Ident.t * string loc * expression) list * class_expr
   | Tcl_constraint of
       class_expr * class_type option * string list * string list * Concr.t
-    (* Visible instance variables, methods and concretes methods *)
+  (* Visible instance variables, methods and concretes methods *)
+  | Tcl_open of override_flag * Path.t * Longident.t loc * Env.t * class_expr
```

Comment:
I think it's the same as for the Texp_open case.  I assumed this could be used by external tools parsing the .cmt files.